app/presenters/publishing_api_presenters/case_study.rb

```diff
@@ -27,6 +27,7 @@ def details
       format_display_type: item.display_type_key,
       first_public_at: first_public_at,
       change_history: item.change_history.as_json,
+      change_note: item.most_recent_change_note,
```

Comment:
Looking at the govt frontend code, I don't believe this is actually used either, and it's not required in the schemas. I think we should remove it here and remove it from the schemas as another PR at some point.